src/datasets/batch_view/sync_summary/reference.md

```diff
@@ -7,26 +7,87 @@
+
+**Q:** Why not include `client_id` in these datasets so that they can be joined on (e.g.) `main_summary`?
+
+**A:** We've had a policy to keep main browser telemetry separate from sync and FxA telemetry. This is in part because FxA `uid`s are ultimately associated with email addresses in the FxA database, and thus a breach of that database in combination with access to telemetry could in theory de-anonymize client-side browser metrics.
+
+## Which apps send sync telemetry? What about Fenix?
+Currently, Firefox for desktop, Firefox for iOS and Firefox for Android (fennec) all have sync implemented, and they all send sync telemetry. Though there are some differences in the telemetry that each application sends, it all ends up in the `sync_summary` and `sync_flat_summary` datasets.
+
+Starting with Fenix, however, sync telemetry will start to be sent through [glean](https://github.com/mozilla-mobile/android-components/tree/master/components/service/glean). This means that, in all likelihood, Fenix sync telemetry will initially be segregated from existing sync telemetry (one reason is that current sync telemetry is on AWS while glean pings are ingested to GCP).
+
+## What's an engine?
+Firefox syncs many different types of browser data and (generally speaking) each one of these data types are synced by their own engine. When the app triggers a "sync" each engine makes their own determination of what needs to be synced (if anything). Many syncs can happen in a day (dozens or more on desktop, usually less on mobile). Telemetry about each sync is logged, and each [sync ping](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/sync-ping.html) (sent once a day, and whenever the user logs in or out of sync) contains information about multiple syncs. The scala code responsible for creating the `sync_summary` dataset unpacks each sync ping into one row per sync. The resulting `engines` field is an array of "engine records": data about how each engine performed during that sync. `sync_flat_summary` further unnests the `engines` field and creates a dataset that is one row per engine record.
+
+Existing engines (`engine_name` in `sync_flat_summary`) are listed below with brief descriptions in cases where their name isn't transparent.
+
+Note that not every device syncs each of these engines. They can be disabled individually and some are off by default.
+
+* `addons`
+* `addresses` mailing addresses e.g. for ecommerce.
+* `bookmarks`
+* `clients` non-user-facing list of the sync account's associated devices
+* `creditcards` this used to be nightly only but was recently removed entirely
+* `extension-storage` webextension-specific storage (ask Lina)?
+* `passwords`
+* `forms` saved values in webforms
+* `logins` deprecated?, use `passwords` (confirm with Lina)
```

Comment:
Hmm, do we still have entries for `logins`? I thought we used `passwords` everywhere...